src/mmrelay/paths.py

```
@@ -0,0 +1,689 @@
+    in MMRELAY_HOME.
+    """
+    return get_plugins_dir() / "core"
+
+
+def _normalize_plugin_type(plugin_type: str | None) -> str | None:
+    if plugin_type is None:
+        return None
+    normalized = plugin_type.strip().lower()
+    if normalized in {"custom", "community", "core"}:
+        return normalized
+    raise ValueError(f"Unknown plugin_type: {plugin_type!r}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)